src/extension.ts

```diff
@@ -221,7 +221,7 @@ class ClientWorkspace {
         };
 
         // Create the language client and start the client.
-        this.lc = new LanguageClient('Rust Language Server', serverOptions, clientOptions);
+        this.lc = new LanguageClient('rust', 'Rust Language Server', serverOptions, clientOptions);
```

Comment:
I feel stupid now :man_facepalming: I remember trying to make `[langId].trace.server` work but couldn't. So that was the missing bit!

Reply:
If it makes you feel any better, I had to step through with the debugger to figure this out!